tools/trustee/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use log::{debug, error};
+
+mod cli;
+mod keys_certs;
+
+#[cfg(feature = "aliases")]
+mod aliases;
+
+#[cfg(feature = "plugins")]
+mod plugins;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init_from_env(env_logger::Env::new().default_filter_or("info"));
+
+    #[cfg(feature = "aliases")]
+    match aliases::match_alias() {
+        Ok(_) => return,
+        Err(e) => {
+            debug!("matching alias failed: {}", e)
+        }
+    }
+
+    #[cfg(feature = "plugins")]
+    match std::env::args().nth(1) {
```

Comment:
I don't really like reading the args manually like this. You are using clap in cli.rs but not here.